### PR TITLE
first commit

### PR DESCRIPTION
**Pull request by carlosissac**
first commit

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,3 @@
-# project1contactmyreps
-project1contactmyreps
+# PROJECT1 : CONTACT MY REPS APP 
+
+Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum
```
Comment (Wanderingtech) [inline README.md:3]: Looks good

File: index.html [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```
Comment (Wanderingtech) [inline index.html]: i don't like this

**Review by Wanderingtech — CHANGES_REQUESTED** [review]
The readme is good. The boilerplate not so much

**Review by Wanderingtech — COMMENTED** [review]
That's a lot of work. Is the responsive nature with all the devices in CSS necessary with materialize being super responsive on it's own?

**Comment by carlosissac** [comment]
I just piggy backed on what I had from my weather dashboard HW.

Materialize is responsive, but there's a few elements that just don't move
the way they should do once I do the resize.

I just copy pasted from my other project and it's working fine.

Tried looking for this message in GIT but i couldn't find it.

Regards.

Issac

**Pull request closed without merging**